### PR TITLE
Remove martin-related toggles from map

### PR DESCRIPTION
**Pull request by tk-vincit**
Resolves HSLdevcom/jore4#[#858](https://github.com/HSLdevcom/jore4-ui/issues/858)

---
This change is [Reviewable](https://reviewable.io/reviews/hsldevcom/jore4-ui/32)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,6 +7,7 @@ COPY ./public ./public
 COPY tsconfig.json next.config.js next-env.d.ts tailwind.config.js postcss.config.js convert-theme-to-ts theme.js ./
 
 ARG NEXT_PUBLIC_GIT_HASH=unknown
+ARG NEXT_PUBLIC_NODE_ENV=production
```
Comment (pallost) [inline Dockerfile:10]: this should be documented to Docker Reference as it is most likely going to be used in the docker-compose environment

**Review by pallost — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/hsldevcom/jore4-ui/32)* status: 0 of 4 files reviewed, 1 unresolved discussion (waiting on @tk-vincit)

___
*[Dockerfile, line 10 at r1](https://reviewable.io/reviews/hsldevcom/jore4-ui/32#-MjnLMX33Z8ZePA-2PEt:-MjnLMX45LKZnCgT5lu1:b-l0iw0u) ([raw file](https://github.com/hsldevcom/jore4-ui/blob/cc29ccbc43b37e62001504a800e942634819c4cb/Dockerfile#L10)):*
> ```Dockerfile
> 
> ARG NEXT_PUBLIC_GIT_HASH=unknown
> ARG NEXT_PUBLIC_NODE_ENV=production
> ```

this should be documented to Docker Reference as it is most likely going to be used in the docker-compose environment

**Review by tk-vincit — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/hsldevcom/jore4-ui/32)* status: 0 of 13 files reviewed, 1 unresolved discussion (waiting on @pallost)

___
*[`Dockerfile` line 10 at r1](https://reviewable.io/reviews/hsldevcom/jore4-ui/32#-MjnLMX33Z8ZePA-2PEt:-N5-eZ801equoiYeQ8im:b9hysvj) ([raw file](https://github.com/hsldevcom/jore4-ui/blob/cc29ccbc43b37e62001504a800e942634819c4cb/Dockerfile#L10)):*
<details><summary><i>Previously, pallost (Tamas Pallos) wrote…</i></summary><blockquote>

this should be documented to Docker Reference as it is most likely going to be used in the docker-compose environment
</blockquote></details>

Comment not anymore relevant after latest changes.

**Review by pallost — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/hsldevcom/jore4-ui/32)* status: 0 of 13 files reviewed, 2 unresolved discussions (waiting on @pallost and @tk-vincit)

___
*[`docker/docker-compose.custom.yml` line 20 at r4](https://reviewable.io/reviews/hsldevcom/jore4-ui/32#-N5-hNEz3Co2hG64itRt:-N5-hNF-1fOnwPy7fnE0:b-80ce3i) ([raw file](https://github.com/hsldevcom/jore4-ui/blob/7982606ae2041f2132537c7cead8122f511ab3af/docker/docker-compose.custom.yml#L20)):*
> ```YAML
> 
>   postgis:
>     # Try to match postgis used in Azure to avoid compatability
> ```

this can also be removed, can't it...?
___
*[`src/components/map/Map.tsx` line 141 at r4](https://reviewable.io/reviews/hsldevcom/jore4-ui/32#-N5-hgNw0z3KCOM4HptG:-N5-hgNw0z3KCOM4HptH:bkyxau4) ([raw file](https://github.com/hsldevcom/jore4-ui/blob/7982606ae2041f2132537c7cead8122f511ab3af/src/components/map/Map.tsx#L141)):*
> ```XML
>                 routes={[
>                   {
>                     iconClassName: 'icon-bus',
> ```

should we also update this icon to e.g. 'icon-route'?

**Review by tk-vincit — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/hsldevcom/jore4-ui/32)* status: 0 of 13 files reviewed, 2 unresolved discussions (waiting on @pallost)

___
*[`docker/docker-compose.custom.yml` line 20 at r4](https://reviewable.io/reviews/hsldevcom/jore4-ui/32#-N5-hNEz3Co2hG64itRt:-N5-jPdn4WRMICUrIG9z:b4jw9gj) ([raw file](https://github.com/hsldevcom/jore4-ui/blob/7982606ae2041f2132537c7cead8122f511ab3af/docker/docker-compose.custom.yml#L20)):*
<details><summary><i>Previously, pallost (Tamas Pallos) wrote…</i></summary><blockquote>

this can also be removed, can't it...?
</blockquote></details>

I think so. Good point...

Done.
___
*[`src/components/map/Map.tsx` line 141 at r4](https://reviewable.io/reviews/hsldevcom/jore4-ui/32#-N5-hgNw0z3KCOM4HptG:-N5-jRuf0mRP_18yDnSg:bvzjdq1) ([raw file](https://github.com/hsldevcom/jore4-ui/blob/7982606ae2041f2132537c7cead8122f511ab3af/src/components/map/Map.tsx#L141)):*
<details><summary><i>Previously, pallost (Tamas Pallos) wrote…</i></summary><blockquote>

should we also update this icon to e.g. 'icon-route'?
</blockquote></details>

Let's check this from design team.

**Pull request merged**